rust/src/editor.rs

```diff
@@ -198,7 +191,26 @@ impl Editor {
         }
     }
 
+    fn delete_forward(&mut self) {
+        if self.view.sel_start == self.view.sel_end {
+            self.move_right();
+        }
+
+        self.delete();
```

Comment:
When we actually implement grapheme clusters, this won't be quite the correct behavior. I'd prefer it to determine `next_grapheme_cluster` and delete that explicitly.